README.md

```diff
@@ -234,6 +234,20 @@ Default: `false`
 
 You can set `hideLabel` to `true` to not render the label element
 
+#### onError

```

Comment:
@cburbank Could you also add these two additional props to the list starting at line 96 (Optional props)?